R/mod-upload-documentation.R

```diff
@@ -25,14 +25,16 @@ upload_documents_ui <- function(id, study_link_human, study_link_animal) {
         shinyjs::disabled(
           fileInput(
             ns("study_doc"),
-            "Upload the study description file"
+            "Upload study description file (.txt, .docx, .md, .pdf, .tex)",
+            accept = c(".txt", ".docx", ".md", ".pdf", ".tex")
```

Comment:
I think we need the MIME types here, not just the file extensions.

Reply:
Oh! That's interesting. It seems to work with the extensions. I changed to the MIME types for these. Thanks for catching that!

Reply:
Oh huh, the documentation says it needs the MIME type but maybe that's not true anymore. I tested your update and it does work as well